### ISSUE TITLE
Radeon HD 5750 fail to start with stak-amd

### ISSUE THREAD

**Issue by Balbec78** [fireice-uk/xmr-stak-amd#22]
If config is

"gpu_threads_conf" : [ { "index" : 0, "intensity" : 64, "worksize" : 8, "affine_to_cpu" : false },]
"platform_index" : 1, 

then

Error CL_INVALID_WORK_GROUP_SIZE when calling clEnqueueNDRangeKernel for kernel 3.

but hashrate shows.

If config is
"gpu_threads_conf" : [ { "index" : 0, "intensity" : 64, "worksize" : 8, "affine_to_cpu" : false },]
"platform_index" : 0, 

then : Error CL_INVALID_DEVICE when calling clCreateCommandQueueWithProperties.

I've tried all AMD drivers this card listed in open cl *.inf files (since 15.12. Crimson it is not)
It takes about 150 h/s on Claymore XMR Miner without any issues, except Claymore's miner is not open sourced.

**Comment by psychocrypt** [comment]
This is fixed in #13 and will be shipped with the next release. If you are using linux you can compile the dev branch (this is the preview for the next release).

```
git clone https://github.com/fireice-uk/xmr-stak-amd.git
cd xmr-stak-amd
cmake .
make install
cd bin
./xmr-stak-amd
```

**Comment by psychocrypt** [comment]
I updated my last post I forgot `git checkout dev`
